tensorboard/webapp/metrics/views/card_renderer/histogram_card_container.ts

```diff
@@ -150,18 +155,30 @@ export class HistogramCardContainer implements CardRenderer, OnInit {
       })
     );
 
-    this.selectedTime$ = this.store.select(getMetricsSelectedTime).pipe(
-      combineLatestWith(this.data$),
-      map(([selectedTime, data]) => {
+    this.steps$ = this.data$.pipe(
+      map((data) => data.map((datum) => datum.step))
+    );
+
+    this.selectedTime$ = combineLatest([
```

Comment:
I thought we had decided to have selectedTime be separate and then that was going to be used to get viewSelectedTime. If you have decided that is not needed or if you plan on doing that in another PR can you please rename this to viewSelectedTime to help clarify that this is not the true selectedTime?

Reply:
Yap I plan to rename it in next PR when doing the comparison. Please wait a bit.